ui/admin/tests/acceptance/managed-groups/create-test.js

```diff
@@ -0,0 +1,160 @@
+    urls.managedGroup = `${urls.managedGroups}/${instances.authMethod.id}`;
+  });
+
+  hooks.afterEach(async function () {
+    const notification = find('.rose-notification');
+    if (notification) {
+      await click('.rose-notification-dismiss');
+    }
+  });
+
+  test('visiting managed groups', async function (assert) {
+    assert.expect(1);
+    await visit(urls.managedGroups);
+    await a11yAudit();
+    assert.equal(currentURL(), urls.managedGroups);
+  });
+
+  test('can create a new managed group', async function (assert) {
+    assert.expect(1);
+    const managedGroupsCount = this.server.db.managedGroups.length;
+    await visit(urls.newManagedGroup);
+    await fillIn('[name="name"]', 'Managed group name');
+    await fillIn('[name="description"]', 'description');
+    await click('form [type="submit"]:not(:disabled)');
+    assert.equal(this.server.db.managedGroups.length, managedGroupsCount + 1);
+  });
+
+  test('User can not create a new managed group without proper authorization', async function (assert) {
+    assert.expect(2);
+    instances.authMethod.authorized_collection_actions['managed-groups'] = [];
+    await visit(urls.authMethod);
+    assert.notOk(
+      instances.authMethod.authorized_collection_actions[
+        'managed-groups'
+      ].includes('create')
+    );
+    assert.notOk(
+      find(`.rose-layout-page-actions [href="${urls.newManagedGroup}"]`)
+    );
+  });
+
+  test('User can navigate to new managed group route with proper authorization', async function (assert) {
+    assert.expect(2);
+    await visit(urls.managedGroups);
+    assert.ok(
+      instances.authMethod.authorized_collection_actions[
+        'managed-groups'
+      ].includes('create')
+    );
+    assert.ok(find(`[href="${urls.newManagedGroup}"]`));
+  });
+
+  test('User cannot navigate to new managed group route without proper authorization', async function (assert) {
+    assert.expect(2);
+    instances.authMethod.authorized_collection_actions['managed-groups'] = [];
+    await visit(urls.managedGroups);
+    assert.notOk(
+      instances.authMethod.authorized_collection_actions[
+        'managed-groups'
+      ].includes('create')
+    );
+    assert.notOk(find(`[href="${urls.newManagedGroup}"]`));
+  });
+
+  test('User can cancel a new managed group creation', async function (assert) {
+    assert.expect(2);
+    const managedGroupsCount = this.server.db.managedGroups.length;
+    await visit(urls.newManagedGroup);
+    await fillIn('[name="name"]', 'Managed group name');
+    await click('form button:not([type="submit"])');
+    assert.equal(this.server.db.managedGroups.length, managedGroupsCount);
+    assert.equal(currentURL(), urls.managedGroups);
+  });
+
+  // Temporary skipping this test because we did not implement mandatory fields in manageg groups
+  test.skip('When user saving a new managed group with invalid fields displays error message', async function (assert) {
```

Comment:
This should not be skipped.  This is a legitimate test of the error notification UX.

Reply:
Updated.